### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by dlockhart**
Just setting up this repo with standard configuration.

I've seeded it with a `createWtrConfig` function just so there's something here and something to test. I do eventually want a shared web-test-runner config like this, but the naming may change before I actually release this.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,73 @@
...
+
+### Testing
+
+```shell
+# lint & run headless unit tests
+npm test
+
+# unit tests only
+npm run test:headless
+```
```
Comment (dlockhart) [inline README.md:34]: We can add documentation for the actual functionality once it solidifies a bit more.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,73 @@
...
+[![NPM version](https://img.shields.io/npm/v/@brightspace-ui/testing.svg)](https://www.npmjs.org/package/@brightspace-ui/testing)
+
+Testing utilities that build upon [@web/test-runner](https://modern-web.dev/docs/test-runner/overview/) but are specifically designed and configured to work with Brightspace components and applications.
+
+## Installation
+
+Install from NPM:
+
+```shell
+npm install @brightspace-ui/foo
```
Comment (svanherk) [inline README.md:12]:
```suggestion
npm install @brightspace-ui/testing
```

File: package.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  "scripts": {
+    "lint": "eslint . --ext .js",
+    "test": "npm run lint && npm run test:headless",
+    "test:headless": "web-test-runner --files \"./test/**/*.test.js\" --node-resolve --playwright"
+  },
+  "author": "D2L Corporation",
+  "license": "Apache-2.0",
+  "devDependencies": {
+    "@web/test-runner": "^0.16",
+    "@web/test-runner-playwright": "^0.10",
```
Comment (svanherk) [inline package.json:16]: Any reason not to move these to `dependencies` from the get-go?

File: .eslintrc.json [hunk 4]
```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "brightspace/node-config",
```
Comment (svanherk) [inline .eslintrc.json:2]: I don't know the differences between all the linting configs offhand - I'm guessing the Lit one doesn't give us anything extra if there's no Lit components?

**Review by svanherk — APPROVED** [review]

**Pull request merged**